package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "ldocap",
+  "version": "0.0.1",
+  "license": "BSD-3-Clause",
+  "description": "Linked Data Capabilities reference implementation.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/digitalbazaar/ldocap.js.git"
```

Comment:
normally this does not include `.git` does it matter?

Reply:
Fix https://github.com/digitalbazaar/ocapld.js/pull/1/commits/881fa7396ed6c099d4d823486922ff1ba10e2ef9